app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/TrackActivity.kt

```diff
@@ -286,6 +289,15 @@ class TrackActivity : SimpleActivity(), PlaybackSpeedListener {
                     finish()
                     activity_track_top_shadow.animate().alpha(0f).start()
                     overridePendingTransition(0, R.anim.slide_down)
+                } else
```

Comment:
oh that is too bad code, stick to the code style used elsewhere please

Reply:
I have reformatted and modified code a little. If this is not what you meant, treat it as a feature request.